kitsune/questions/jinja2/questions/includes/question_editing_frame.html

```diff
@@ -24,7 +24,9 @@
       {{ progress_bar(current_step, product_key=current_product.key) }}
     {% endif %}
 
-    {{ scam_banner() }}
+    {% if current_step > 1 and not zendesk_form %}
```

Comment:
Similar to the other comment, maybe it makes sense to pass the product object here and extract the info from there